subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -417,6 +455,14 @@ int aws_fota_mqtt_evt_handler(struct mqtt_client *const client,
 		if (evt->result != 0) {
 			return 0;
 		}
+		if (evt->param.suback.message_id == SUBSCRIBE_NOTIFY_NEXT) {
```

Comment:
nit: newline after `}`

Reply:
Hm, or was that upstream that someone said that. Unsure, just leave it as is if its like this other places.